packages/js/package.json

```diff
@@ -4,6 +4,7 @@
   "license": "GPL-3.0",
   "private": true,
   "scripts": {
+    "build": "cd ../.. && wp-scripts build --config config/webpack/webpack.config.js",
```

Comment:
If we build here, we should probably move/copy the .babelrc file from the root to this package.

Reply:
I wanna leave that for a separate change, to prevent scope creep of this PR.